go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -120,14 +202,15 @@ def local_replacements
             end
         end
 
-        def prepared_go_mod_content
-          content = updated_go_mod_content
-          local_replacements.reduce(content) do |body, (path, stub_path)|
-            body.sub(path, stub_path)
+        def substitute_all(file, substitutions)
+          substitutions.reduce(file) do |text, (a, b)|
+            text.sub(a, b)
           end
         end
 
-        def handle_subprocess_error(path, stderr)
+        def handle_subprocess_error(stderr)
+          stderr = stderr.gsub(Dir.getwd, "")
```

Comment:
The `path` argument was always intended to be the cwd, and was provided so we can strip it from the error message. It's simpler for the caller if we just look up the cwd directly.